munch.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Akash Kakkar (akashk07)",
+			"oem": "Poco",
+			"device": "Poco F4",
+			"filename": "crDroidAndroid-14.0-20231124-munch-v10.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/munch/10.x/crDroidAndroid-14.0-20231124-munch-v10.0-BETA.zip/download",
+			"timestamp": 1700768433,
+			"md5": "3b42a3a9d2e929662bf3783b4a957542",
+			"sha256": "be10d9ef627900bacac69dbf78f5684b123e1017294b63af2c8a21ceb281b973",
+			"size": 1612181517,
+			"version": "10.0",
+			"buildtype": "Monthly",
+			"forum": "https://forum.xda-developers.com/t/4492155",
```

Comment:
please link forum download url to website, not sourceforge

Reply:
Done, updated the download link in the forum.
Please check